### DISCUSSION TITLE
How to specify interactsh server ports

### DISCUSSION THREAD

**Discussion by 0xcrypto** [projectdiscovery/nuclei#3989] [Q&A]
I am running interactsh server in a restricted environment and cannot open ports 53, 80 and similar. So I am using custom ports like 8080, 8053, 8443 etc. for all services. I am using `iserver` flag to specify interactsh server but  I couldn't find any flag in nuclei to specify the ports.  So, how do I specify them in nuclei?

**Comment by MetzinAround** [comment]
Thanks for the question, @0xcrypto. I'm gonna tag @olearycrew and see if he has an idea about how to specify a port when using interactsh in nuclei.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
